scipy/stats/morestats.py

```diff
@@ -2441,6 +2441,8 @@ def levene(*args, center='median', proportiontocut=0.05):
     return LeveneResult(W, pval)
 
 
+@np.deprecate(new_name='binomtest',
+              message='`binom_test` will be removed from SciPy 1.8.')
```

Comment:
@rlucas7 @WarrenWeckesser Should this still be 1.8, given that this didn't make it into 1.6?
Also, did we send a message to the mailing list? I'm just thinking that we should do that before putting this in.

Reply:
@mdhaber, I'll change that to 1.9.  And I agree that this should be brought up on the mailing list before we merge this.

Reply:
ok, I'll hold off merging then :)

Reply:
The code was updated to say 1.9, and I just sent an email about this to the mailing list.